.github/workflows/build_and_test.yml

```diff
@@ -21,18 +21,6 @@ jobs:
         with:
           java-version: 1.11
 
-      - name: Install curl Linux
```

Comment:
Why we don't need this anymore?

Reply:
This was needed, when we were trying to download test resoruces (`jar`s) in test themselves, and now we are doing it via gradle task